### DISCUSSION TITLE
Issue When Asembling Frame with TouchScreen to Case

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#44] [Q&A] [answered]
I've reached the point where all components are attached to the frame and I've tried to put the frame into the case.

The end that contains the Firebeetle and on/off switch is not flush with the case. It looks like the TouchScreen base is protruding  a skosh. The other end containing the rotary encoder is fine.

I will take the frame off of the top and determine what I can do to correct the mismatch. 

 [ **skosh** - a small amount : [BIT] (https://www.merriam-webster.com/dictionary/bit), [SMIDGEN] (https://www.merriam-webster.com/dictionary/smidgen)] [n. it's from Japanese "sukoshi" meaning "small". The first U is often unvoiced, so the English imitation of it is to drop it altogether, along with thefinal vowel. So it's a "skosh".]

**Comment by daveclifford** [comment, marked as answer]
I looked closely at the frame and the cross member closest to the firebeetle2/switch may be slightly too thick and causing the frame edge not to fit to the case. I need to take the components off and fine file the cross member a MM. 

Dave

**Reply by daveclifford** [reply]
Two observations:

1. One of the 2 screws holding the on/off switch to the frame **was not** screwed in all the way, causing the frame to produde out from the end.
2. The 3-D print is pretty good, but the frame is slightly warped, such that the side closest to the USB-C connection is not flat. I may correct the mis-alignment by  slightly sanding the post on one side.

**Reply by alex-code** [reply]
The design is pretty tight so any fitment issues are likely from 3D printing tolerances.
Not much that can be done apart from light trimming and fettling.

**Reply by daveclifford** [reply]
Thanks for the reponse - Yep - I can live with that - The case back is flush - the front is off by a hair. I can live with that, since most won't look too closely. They will be looking at the screen and turning the encoder knob.
